tests/authentication.rs

```diff
@@ -34,7 +34,6 @@ mod authentication_tests {
             .env_clear()
             .envs(Environment::inherit().compile())
```

Comment:
A lot of repeated options in many files. Would it be worthwile to centralize a basic cargo creation somewhere (and let the caller pass the rest of options  + spawn)